RangeRequest.html

```diff
@@ -4,10 +4,9 @@
   <meta content="width=device-width, initial-scale=1, shrink-to-fit=no" name="viewport">
```

Comment:
(review based on .bs while reading the html; no review comments on generated file)